MapboxCoreNavigation/MBXAccounts+CoreNavigationAdditions.m

```diff
@@ -0,0 +1,14 @@
+#import "MBXAccounts+CoreNavigationAdditions.h"
+
+@implementation MBXAccounts (CoreNavigationAdditions)
+
++ (void)load {
+    NSString *billingMethodValue = [NSBundle.mainBundle objectForInfoDictionaryKey:@"MBXNavigationBillingMethod"];
+    MBXAccountsSKUID skuIdentifier = MBXAccountsSKUIDNavigationSession;
+    if ([billingMethodValue isEqualToString:@"mau"]) {
```

Comment:
This logic needs to be revised so that `mau` is the default and there’s an option to opt out of a SKU altogether for accounting based on API calls.